Postman/SurveyInviteRespondents.postman_collection.json

```diff
@@ -0,0 +1,308 @@
+					"raw": "{{origin}}/v1/Surveys/InviteRespondents/SurveysInvitationStatus",
+					"host": [
+						"{{origin}}"
+					],
+					"path": [
+						"v1",
+						"Surveys",
+						"InviteRespondents",
+						"SurveysInvitationStatus"
+					]
+				},
+				"description": "This method gets invitation status."
+			},
+			"response": []
+		},
+		{
+			"name": "Get Invitation Batches status",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"const validateProperties = (item) => {",
+							"    return item.SurveyId !== undefined ",
+							"    && item.BatchName !== undefined ",
+							"    && item.ScheduledFor !== undefined",
+							"    && item.Status !== undefined",
+							"    && item.TotalCount !== undefined    ",
+							"    && item.ScheduledCount !== undefined",
+							"    && item.PendingCount !== undefined",
+							"    && item.NotSentCount !== undefined",
+							"    && item.ErrorCount !== undefined",
+							"    && item.SentCount !== undefined",
+							"    && item.OpenedCount !== undefined",
+							"    && item.ClickedCount !== undefined",
+							"    && item.UnsubscribedCount !== undefined",
+							"    && item.AbuseReportCount !== undefined",
+							"    && item.UnknownCount !== undefined",
+							"    ;",
+							"}",
+							"",
+							"pm.test(\"Response contains all fields\", function () {",
+							"    var invitations = pm.response.json();",
+							"    pm.expect(invitations.every(validateProperties)).to.be.true;  ",
+							"",
+							"    if (invitations.length > 0)",
+							"    {",
+							"        var lastIndex = invitations.length - 1;",
+							"        pm.environment.set(\"BatchName\", invitations[lastIndex].BatchName);",
+							"    }  ",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					},
+					{
+						"key": "Authorization",
+						"value": "Basic {{AuthenticationToken}}"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\r\n  \"Script\": \"*MERGE \\\"destinationlist\\\"\\r\\n*QUESTION 1 *CODES 61L1 *LIST \\\"destination\\\"\\r\\nWhat is your destination (postman)?\\r\\n*END\",\r\n  \"FileName\": \"destination\",\r\n  \"UnfixedIsOk\": true\r\n}"
+				},
+				"url": {
+					"raw": "{{origin}}/v1/Surveys/{{SurveyId}}/InviteRespondents/SurveyBatchesStatus",
+					"host": [
+						"{{origin}}"
+					],
+					"path": [
+						"v1",
+						"Surveys",
+						"{{SurveyId}}",
+						"InviteRespondents",
+						"SurveyBatchesStatus"
+					]
+				},
+				"description": "This method gets invitation batches status."
+			},
+			"response": []
+		},
+		{
+			"name": "Get Invitation batch status",
```

Comment:
This one has (almost) the same name as the previous one, maybe the difference should be indicated in the name.